docs/crd.adoc

```diff
@@ -4,7 +4,7 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-09T03-21-45Z]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-24T23-19-33Z]
```

Comment:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-24T23-19-33Z]
```

No change is needed to this version yet - let's keep the default with the previous version for now - we will move to a newer release this week instead.